packages/app/src/cli/services/function/binaries.test.ts

```diff
@@ -313,3 +320,41 @@ describe('wasm-opt', () => {
     await expect(fileExists(wasmOpt.path)).resolves.toBeTruthy()
   })
 })
+
+describe('trampoline', () => {
+  const trampoline = trampolineBinary()
+
+  test('properties are set correctly', () => {
+    expect(trampoline.name).toBe('shopify-function-trampoline')
+    expect(trampoline.version).match(/v\d.\d.\d$/)
+    if (process.platform === 'win32') {
+      expect(trampoline.path).toMatch(/(\/|\\)shopify-function-trampoline\.exe$/)
+    } else {
+      expect(trampoline.path).toMatch(/(\/|\\)shopify-function-trampoline$/)
+    }
+  })
+
+  test('downloadUrl returns the correct URL', () => {
+    // When
+    const url = trampoline.downloadUrl('darwin', 'x64')
+
+    // Then
+    expect(url).toMatch(
+      /https:\/\/github.com\/Shopify\/shopify-function-wasm-api\/releases\/download\/shopify_function_trampoline\/v1.0.0\/shopify-function-trampoline-x86_64-macos-v\d\.\d\.\d.gz/,
```

Comment:
This assertion will fail when we update the trampoline version. Do you think we should have it use `TRAMPOLINE_VERSION` in there to ensure it stays in sync automatically?